readme.md

```diff
@@ -41,7 +41,7 @@ var theBody = document.querySelectorAll("body");
 theBody.style.backgroundColor = "red";
 ```
 
-> Your answer...
+> background and color should be separated with a -.
 
```

Comment:
Here the issue is that `querySelectorAll` returns an array of DOM object, even if there is only one as is the case with `body`. There are a few ways to write this out correctly...

```js
document.body
document.querySelectorall("body")[0]
document.querySelector("body")
```